### PR TITLE
Viewer types

### PR DESCRIPTION
**Pull request by seanmtracey**
Added functionality that will let us determine and filter the types of viewers on the admin screen. Should be handy for debugging, and potentially logging the usage of each of the different types #51

### CODE REVIEW

File: client/viewer/js/viewer.js [hunk 1]
```diff
@@ -116,6 +116,22 @@ const __client = (function(moment) {
 		document.title = name + ' : FT Screens';
 	}
 
+	function checkIfChromeExtension(){
+
+		if(window.chrome !== undefined){
+
+			if(window.chrome.runtime.id !== "web"){
```
Comment (seanmtracey) [inline client/viewer/js/viewer.js:123]: Thinking about it, this should be more robust

File: views/partials/screen.handlebars [hunk 2]
```diff
@@ -1,4 +1,4 @@
-	<tr class='screen' id='screen-{{{id}}}' data-id='{{id}}' data-filter='{{toLower id}} {{toLower name}}{{#tags}} {{toLower .}}{{/tags}}'>
+	<tr class='screen' id='screen-{{{id}}}' data-id='{{id}}' {{#if type}}data-viewer-type="{{type}}"{{/if}} data-filter='{{toLower id}} {{#if type}}type:{{type}}{{/if}} {{toLower name}}{{#tags}} {{toLower .}}{{/tags}}'>
```
Comment (JakeChampion) [inline views/partials/screen.handlebars:1]: I don't think the filter command uses fuzzy searching, meaning that I would have to write the type of screen out first and then the name in order to locate a specific screen.

**Comment by JakeChampion** [comment]
The viewer code in this repository is legacy code I believe, soon to be replaced with -- https://github.com/ftlabs/screens-viewer

Analytics for each version of the viewer application should live in the viewer codebases (Chrome-extension, Electron, website). 

It may be safer to let the viewers say what type of viewer they are themselves rather than infer it through properties.

**Comment by seanmtracey** [comment]
That makes sense. Also, you're correct about the input filtering.

**Pull request closed without merging**